src/styles/_global.scss

```diff
@@ -108,8 +108,8 @@ textarea {
 // Outer elements -- <html> and <body>
 html,
 body {
-	inline-size: 100%;
-	block-size: 100%;
+	block-size: 100vh;
+	block-size: 100dvh;
```

Comment:
Double block size?

Reply:
this is intended: on devices where dynamic viewport height is supported we want to use that, but if it's not supported that rule will be ignored and regular viewport height will be used instead.

Reply:
Only IE and Opera don't support this but I'm a bit surprised Svelte or Sass don't take care of that.

Reply:
It's not just which browser, but what version. All of these are relatively new versions of the evergreen browsers, and it's only at 75% global support.
![image](https://user-images.githubusercontent.com/65342367/213912390-a46fd422-2694-4b9c-931f-08e763e39227.png)

Reply:
Also, neither CSS nor JS frameworks like Sass and Svelte should interfere in your styles. That's what postcss is for.

Reply:
Oh didn't thought it was that recent. Still surprised we need to think of this ourselfes given tools exist that handle fallbacks like that.

Reply:
I would honestly expect my CSS transform to also provide a fallback for stuff like that. IIRC Webpack also does this though I haven't used that in ages.